vtex/omnishipping
# CHUI-79: Remove /utils folder in favor of shipping-manager's exported functions

#### What is the purpose of this pull request?
The `/utils` folder is duplicated in shipping-manager and omnishipping. This PR deletes `/utils` folder of this repository and replaces all uses by the functions exported by shipping-manager.

This depends on https://github.com/vtex/shipping-manager/pull/41.

#### How should this be manually tested?
Since this is a big code change, I'm relying on our E2E tests to check for regressions. I've linked these changes in the `beta` workspace of our tests accounts.

#### Types of changes
